mod.ts

```diff
@@ -18,55 +19,13 @@ async function fetchAndParseHTML(htmlOrUrl: string) {
   return parse(html)
 }
 
-type MaybeOpenGraphTags = {
-  title?: string
-  type?: string
-  description?: string
-  site_name?: string
-  locale?: string | { alternate?: string; content: string }
-  image?:
-    | string
-    | {
-        url?: string
-        secure_url?: string
-        type?: string
-        width?: string
-        height?: string
-        alt?: string
-        content: string
-      }
-  url?: string
-  determiner?: string
-  [extraKeys: string]: unknown
-}
-
-type MaybeTwitterTags = {
-  title?: string
-  card?: string
-  site?: string
-  description?: string
-  image?: string
-  image_alt?: string
-  player?: string
-  player_width?: string
-  player_height?: string
-  player_stream?: string
-  app_name?: string
-  app_id_googleplay?: string
-  app_id_iphone?: string
-  app_url_googleplay?: string
-  app_url_iphone?: string
-  app_country?: string
-  [extraKeys: string]: unknown
-}
-
 /**
  * Extracts all meta tags from a given HTML string or URL
  * @param htmlOrUrl {string} HTML string or URL to parse, if URL is given it will be fetched
  * @param prefix {string} Optional prefix to filter meta tags, e.g. 'og:' or 'twitter:'
  */
-export async function getMetaTags(htmlOrUrl: string, prefix = '') {
-  const returnObj: Record<string, unknown> = {}
+export async function extractMetaTags(htmlOrUrl: string, prefix: string = '') {
```

Comment:
There's no need to put the `string` comment in here since it's a trivial type.

```suggestion
export async function extractMetaTags(htmlOrUrl: string, prefix = '') {
```

Reply:
it's because the Deno linter says: "JSDoc types may be moved to TypeScript types."

Reply:
Ok makes sense, I'd prefer that in the TS types other than in the doc, I'll resolve this